gen-models/gen-models.capnp

```diff
@@ -208,8 +208,11 @@ struct ManifestOperation {
 struct Manifest {
   manifestVersion @0 :Text;
   branchName @1 :Text;
-  endHash @2 :List(UInt8);
-  operations @3 :List(ManifestOperation);
+  endHash :union {
```

Comment:
when pulling into a fresh repository, no hashes exist so this was made optional.

Reply:
:+1: